modules/ROOT/pages/6.0-release-notes-parsing-and-serialisation.adoc

```diff
@@ -0,0 +1,24 @@
+= TinyMCE 6.0 parsing and serialisation changes
+:navtitle: TinyMCE 6.0 parsing and serialisation changes
+:description: TinyMCE 6.0 parsing and serialisation changes
+:keywords: releasenotes, parsing, serialisation
+
+This section documents the single change made to {productname} 6.0 parsing and serialisation.
+
+* xref:parsing-and-serialisation[Parsing and serialisation]
+* xref:changed-parsing-and-serialisation[Changed parsing and serialisation]
+
+// tag::parsing-and-serialisation[]
+[[parsing-and-serialisation]]
+== Parsing and serialisation
+
+
+[[changed-parsing-and-serialisation]]
+=== Changed parsing and serialisation
+
```

Comment:
Again, having so many headers for one subsection feels weird to me. We might be using them though?